grpc/binlog/v1alpha/binarylog.proto

```diff
@@ -61,6 +65,17 @@ message GrpcLogEntry {
   // Peer address information, will only be recorded in SEND_INITIAL_METADATA
   // and RECV_INITIAL_METADATA entries.
   Peer peer = 6;
+
+  // The method name. Always logged for every entry.
+  string full_method_name = 8;
```

Comment:
Let's just call this `method_name`.  The `full_` prefix doesn't add any value, since gRPC always uses absolute method names.

Reply:
Done